### ISSUE TITLE
add a tag (maybe v0.1)

### ISSUE THREAD

**Issue by MajdT51** [edgraaff/quorum-rpc-proxy#3]
Could you please add a tag (which I can depends on when I clone your repository in my script), in case you updated or changed some of the features in the master branch.

https://git-scm.com/book/en/v2/Git-Basics-Tagging

**Comment by edgraaff** [comment]
Hiya!

I could of course .. but for a forked repository this isn't necessary? Just add the original repo (which is mine in this case) as another remote in your cloned repo. To get updates, fetch from all remotes and merge 'my' master with 'your' master.
Still, if you need a tag, I can add it just fine. It's more that I'm curious what for :)

**Comment by MajdT51** [comment]
yes please add the tag anyway. I want to use the proxy inside a docker container with other components (not for production purposes) :)

@edgraaff
